bigbluebutton-html5/imports/ui/components/actions-bar/actions-dropdown/component.jsx

```diff
@@ -66,6 +66,34 @@ class ActionsDropdown extends Component {
     this.props.mountModal(<PresentationUploaderContainer />);
   }
 
+  getAvailableActions() {
+    const {
+      intl,
+      handleShareScreen,
+      handleUnshareScreen,
+      isVideoBroadcasting,
+    } = this.props;
+
+    return _.compact([
+      (<DropdownListItem
+        icon="presentation"
+        label={intl.formatMessage(intlMessages.presentationLabel)}
+        description={intl.formatMessage(intlMessages.presentationDesc)}
+        key={_.uniqueId('action-item-')}
+        onClick={this.handlePresentationClick}
+      />),
+      (Meteor.settings.public.kurento.enableScreensharing ?
+        <DropdownListItem
+          icon="desktop"
+          label={intl.formatMessage(isVideoBroadcasting ? intlMessages.stopDesktopShareLabel : intlMessages.desktopShareLabel)}
+          description={intl.formatMessage(isVideoBroadcasting ? intlMessages.stopDesktopShareDesc : intlMessages.desktopShareDesc)}
+          key={_.uniqueId('action-item-')}
```

Comment:
The part with the key is something that wasn't right in that example..
Could you generate the keys in the componentWillMount and then just use them here?
e.g 

```
componentWillMount() {
  this.key1 = _.uniqueId('action-item-');
  this.key2 = _.uniqueId('action-item-');
}
.....
<DropdownListItem
  ...
  key={this.key1}
  ...
/>
```

Just give them the names, please.

Reply:
Ohh, ok. Fixed.